chain/gssmr/config.toml

```diff
@@ -40,6 +40,8 @@ port = 8545
 host = "localhost"
 modules = ["system", "author", "chain", "state", "rpc", "grandpa", "offchain", "childstate", "syncstate", "payment"]
 ws-port = 8546
+ws-unsafe-external = false
+unsafe-external = false
```

Comment:
are these not false by default?

Reply:
I put to be explicit but definitely we don't need